src/tasks/DailyTask.py

```
@@ -458,44 +458,76 @@ def gongongqu(self):
             if self.config.get("自主循环"):
                 self.auto_loop()
 
+    def _auto_loop_step_with_retry(self, step_num, match, box=None, time_out=5, after_sleep=2, **kwargs):
+        """
+        点击OCR匹配元素，等待点击特征在 3s 内消失。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)